db/migrations/versions/006_replace_enums_with_strings.py

```diff
@@ -0,0 +1,232 @@
+"""empty message
+
+Revision ID: 720d88c96ef6
+Revises: 005_relax_proj_prog_nulls
+Create Date: 2023-07-03 12:10:30.753706
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "720d88c96ef6"
+down_revision = "005_relax_proj_prog_nulls"
+branch_labels = None
+depends_on = None
```

Comment:
`revision` and `Revision ID` should reflect the custom filename you've given

Reply:
done

Reply:
they shouldn't have the `.py` bit - should be like line 14 `down_revision = "005_relax_proj_prog_nulls"`